local/notebooks/analysis.py

```diff
@@ -11,21 +11,35 @@
 def run_state_dict(datasets, label, mask, save_fig):
     """Runs an analysis across state for multiple dictionaries
     
-    datasets: list of dict of 4d arrays
+    datasets: list of dict
+        each list entry is a state
+            0 - Trial
+            1 - Rest
+        each dict entry is a band with 3d array of subject channel feature 
     label: str
     eeg_dat_info: str
     pos: int
     save_fig: boolean
     """
 
     bands = datasets[0].keys()
-    feats = ["CFS", "AMPS", "BWS"]
+    # extracting the band names from one of the lists
+    # "theta, alpha, beta"
+
+    feats = ["CFS", "PWS", "BWS"]
+    # creating a list of the feature names 
 
     corr_dicts = []
 
     for band in bands:
+    # This loop goes through each of the bands individual bands
+    # Resulting in a [n_subjects, num_blocks, n_channels, n_feats] 
 
         curr_data = [dataset[band] for dataset in datasets]
+        # curr_data is a list of [n_subjects, num_blocks, n_channels, n_feats] for both states
```

Comment:
For consistency, when indicating the # of something, always `n_thing`.
So, `num_blocks` should be `n_blocks` (here and elsewhere).